tests/keras/utils/np_utils_test.py

```diff
@@ -28,6 +29,38 @@ def test_to_categorical():
         # Get original labels back from one hots
         assert np.all(np.argmax(one_hot, -1).reshape(label.shape) == label)
 
+def test_mxnet_image_channels_conversion():
+    shapes = [
+        (10, 32, 32, 3),
+        (10, 3, 32, 32),
+        (10, 24, 128, 128, 1),
+        (10, 1, 24, 128, 128)
+    ]
+
+    expected_shapes = [
+        (10, 3, 32, 32),
+        (10, 3, 32, 32),
+        (10, 1, 24, 128, 128),
+        (10, 1, 24, 128, 128)
+    ]
+
+    inputs = [np.random.randint(0, 255, shape) for shape in shapes]
```

Comment:
negative test cases?
Ex: Invalid data_format, Invalid Tensor dimension etc

Reply:
Already handled Invalid Tensor dimension use cases where it throws a ValueError if the tensor dimension is incorrect.

Can you please elaborate what do mean by `invalid_data_format` means. Are your referring to having a use case to check `data_format` as `channels_first` or `channels_last`?

Thank-You.